src/poetry/utils/env/virtual_env.py

```diff
@@ -133,3 +136,14 @@ def temp_environ(self) -> Iterator[None]:
 
     def _updated_path(self) -> str:
         return os.pathsep.join([str(self._bin_dir), os.environ.get("PATH", "")])
+
+    @cached_property
+    def includes_system_site_packages(self) -> bool:
+        pyvenv_cfg = self._path / "pyvenv.cfg"
+        return "include-system-site-packages = true" in pyvenv_cfg.read_text()
```

Comment:
This line seems a bit fragile so I wondered if `pyvenv.cfg` is some well defined format. At first, I did not find a specification, but the `site` module being more robust considering spaces and case-sensitivity:

https://github.com/python/cpython/blob/042aa88bcc6541cb8b312f1119452f7a58a5b4df/Lib/site.py#L515-L522

Then, I found PEP 405, which describes how the `site` module parses it so I think we should be more robust, too.